common.cxx

```diff
@@ -696,19 +696,24 @@ void SVD_solve(Matrix<>& M,
     Timer tSVD_solve("SVD_solve");
     tSVD_solve.start();
     // Perform SVD
-    Matrix<> U,VT;
+    /**Matrix<> U,VT;
     Vector<> s;
-    // if (random) {
-    //     randomized_svd(S, U, s, VT, S.ncol, 1);
-    // } else {
-    S.svd(U,s,VT,S.ncol);        
```

Comment:
I suggest keep both versions (SVD and Cholesky), and add an argument to the run.cxx to choose which version to use